### ISSUE TITLE
Resource not found

### ISSUE THREAD

**Issue by DavidFerreira1002** [FedericoRollo/followme#3]
After I ran catkin_make and sourced the workspace, I did roslaunch followme followme.launch and it gave me an error like this:

Resource not found: target_estimation
ROS path [0]=/opt/ros/noetic/share/ros
ROS path [1]=/home/david/Projects/FollowMe/src
ROS path [2]=/home/david/Projects/RC_Project/src
ROS path [3]=/opt/ros/noetic/share
The traceback for the exception was written to the log file

So I went looking in the followme.launch file and I found this:

--

    <include file="$(find target_estimation)/launch/target_node_spawn.launch">
        <arg name="token" value="target"/>
        <arg name="timeout" value="2.5"/>
        <arg name="output" value="log"/>
    </include>
--

the only place that has "target_estimation" is on the same launch file here as a namespace:

--

    <rosparam command="load" file="$(find followme)/kf_models/model_uniform_acceleration_params.yaml" ns="target_estimation"/>
--

Im not sure if there was more code on your side, maybe a robot specific code or something else, but I cant find that ("target_estimation") in the followme package nor the MMT, ai_utils and yolact.

Am I missing something? 
And should I be doing this type of questions here? Still new to this, thanks
PS: I cant seem to assign this to you Federico

**Comment by FedericoRollo** [comment]
yes, here is the right place to ask questions! If the repo in which you open the issue is on my personal profile I receive a notification. If the repo is on an organization I don't receive any issue message If the issue is not assigned to me. 

Anyway, you are right, I forgot a dependency in the readme file. You should clone the following repository in the src directory of your catkin workspace and re-compile. Then it should find the dependency.

**Comment by DavidFerreira1002** [comment]
Got it, I also need this import from track_person_node.py:

"from camera_utils.from2Dto3D import compute_centroids"

where does it come from? thanks

**Comment by FedericoRollo** [comment]
Another missing dependency... Sorry but this was an early package, I have an updated package but I cannot disclose it without permission now. 

Anyway, install [camera_utils](https://github.com/IASRobolab/camera_utils) as for the other packages. Follow the installation section.

**Comment by DavidFerreira1002** [comment]
I'm having a problem that doesn't have to do directly with followme, its with CUDA, I am using VMware Workstation to have a Linux Ubuntu 20.04 virtual machine to run ROS Noetic, I tried installing CUDA and found out that isn't going to work in a VM and that I should get VMware SVGA for that, I tried to install it and its just been errors after errors after errors, dependencies are killing me.

What is your setup? Do you use a VM?

**Comment by FedericoRollo** [comment]
I'm not able to help you with virtual machines... sorry.
CUDA has some hardware interface that maybe the virtual machine is not able to simulate.

I've often used an Ubuntu 20.04 machine and I installed the code directly on the system. 
An alternative could be to use the docker. I've used it especially on Ubuntu 18 because AI modules request python 3 and there are incompatibilities with ROS melodic. For example on some robots there is only Ubuntu 18 and you are constrained to use Dockers. they are a good alternative if you are using other Ubuntu versions.
